specification/glossary.md

```diff
@@ -2,10 +2,130 @@
+
+## Commitment-Based Discount
+
+Also known as Commitment Discount, this is a pricing strategy or incentive offered by a service provider to encourage customers to commit to a certain level of usage or purchase over a specified period. Offerings vary in specificity of commitment, penalties for underutilization, and possible remedies.
+
+## CSP
+
+Cloud Service Provider (e.g. Amazon Web Services, Microsoft Azure, Google Cloud Platform).
+
+## Dimension
+
+Specification-defined categorical attribute that provides context or categorization to billing data.
+
+## Effective Cost
+
+A comprehensive and realistic measure of the true cost of a service, inclusive of all reduced rates, discounts, and impacts of amortization.
+
+## Finalized Tag
+
+A tag with one tag value chosen from a set of possible tag values after being processed by a set of provider-defined or user-defined rules.
+
+## FOCUS
+
+FinOps Cost and Usage Specification - An effort to build an open specification for cloud billing data.
+
+## Interruptible
+
+Category of compute resources that can be paused or terminated by the CSP within certain criteria, often available for discounted pricing when compared to the equivalent non-interruptible resource.
+
+## List Unit Price
+
+Provider published unit price for a single pricing measurement unit.
+
+## Metric
+
+A quantifiable, standardized measure used to assess, evaluate, or track the performance, quality, or characteristics of a system, process, or service.
+
+## MSP
+
+Managed Service Provider - A company or organization that provides outsourced management and support of a range of IT services, such as network infrastructure, cybersecurity, cloud computing, and more.
+
+## On-Demand
+
+A term that describes a service that is available and provided immediately or as needed, without requiring a pre-scheduled appointment or prior arrangement. In Cloud Computing, virtual machines can be created and terminated as needed, i.e. on demand.
+
+## Practitioner
+
+Any individual whose charter is to enhance business value from cloud infrastructure, by balancing innovation, velocity, and cost efficiency.
+
+## Provider
+
+An entity that delivers services to consumers or clients i.e. a company that offers cloud computing, managed IT services, web hosting, or other technology-related services.
+
+## Price list
+
+Comprehensive list of prices for offerings from a CSP, published by the CSP.
+
+## Resource
+
+An provisionable infrastructure component that is available to a customer in a cloud environement e.g. virtual machine, storage volume etc.
+
+## Row
+
+An entry in a billing data set representing the consumption of a resource or other measurable activity.
+
+## Service
+
+A capability defined and offered by cloud service providers to satisfy certain needs, often through activation of resources e.g. the capability of creating a virtual machine or a storage volume are services.
+
+## SKU
```

Comment:
```suggestion
## Sku
```

I think this is the form we use the specs?

Reply:
No... Display names will be `SKU`, column IDs will be `Sku`.  [Here](https://finopsfoundation.slack.com/archives/C053CNCRVCL/p1699049826828449) is a link to the poll and resolution.